internal/playwright/config.go

```diff
@@ -77,6 +77,7 @@ type SuiteConfig struct {
 
 	// Fields appeared in v1.12+
 	Headed        bool   `yaml:"headed,omitempty" json:"headed,omitempty"`
+	Headless      bool   `yaml:"headless,omitempty" json:"headless,omitempty"`
```

Comment:
If we're adding `headless` something needs to be done about the `headed` config then.

Reply:
good point! I'll mark it as deprecated